akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/package.scala

```diff
@@ -17,15 +17,15 @@ package object scaladsl {
    * Enable these extension methods with:
    *
    * {{{
-   * import akka.actor.typed.scaladsl.LoggerOps
-   * }}}
+   *    * }}}
    * or
    * {{{
    *import akka.actor.typed.scaladsl._
    * }}}
    *
    * @param log the underlying [[org.slf4j.Logger]]
    */
+  @deprecated("Not needed in 2.13 and later", "2.10.0")
```

Comment:
Isn't deprecating the class enough (and implicitly means all methods are deprecated as well)

Reply:
I was not sure. If you suggest, I can use that single-deprecation approach.

Reply:
Just the class is enough